.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Run Extension",
+			"type": "extensionHost",
+			"request": "launch",
+			"args": [
+				"--extensionDevelopmentPath=${workspaceFolder}",
+				"${workspaceFolder}/../examples/test-project"
```

Comment:
Specify folder in which debug of extension will be started. It is necessary that there is some folder to run `Install hermione` action

Reply:
Agree, it will be more convenient